### DISCUSSION TITLE
Syntax highlight for beatmap searches

### DISCUSSION THREAD

**Discussion by AuracleTech** [ppy/osu#17338] [Ideas]
Apply a style to the search terms so we can know when it's valid.
A small ( i ) with the keys listed could be nice too!
Also a playcount argument could be added

![image](https://user-images.githubusercontent.com/40531871/159104181-78f9a742-31ea-482b-bfa0-b2d332a46380.png)

**Comment by peppy** [comment]
I think we'd sooner add a UI for these filters rather than adding syntax highlighting. A user shouldn't need to memorise these in the first place.

If we were to add better display in the text line, it would be done similar to discord's search, where valid keywords get converted into pills or something.
